demo/universal/native/lib/Counter.re

```diff
@@ -1,10 +1,19 @@
-[@warning "-27"];
+let make = (~initial, ~onClick as [@browser_only] onClick) => {
+  let (count, [@browser_only] setCount) = RR.useStateValue(initial);
 
-let make = (~initial) => {
-  let (state, setCount) = RR.useStateValue(initial);
+  switch%platform (Runtime.platform) {
+  | Server => print_endline("This prints to the server terminal")
+  | Client => print_endline("This prints to the console")
+  };
```

Comment:
Is this needed?

Reply:
@jchavarri This piece of code was present before, but @davesnx removed it, and on the merge, I didn't notice it. Going to remove it.